app/views/tags/aggregatedMetrics.scala.html

```diff
@@ -0,0 +1,33 @@
+@*
+* Copyright 2016 LinkedIn Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy of
+* the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations under
+* the License.
+*@
+
+@(result: models.AppResult)
+@import com.linkedin.drelephant.util.Utils;
+
+
+@if(!result.jobType.equals("Spark")) {
```

Comment:
Add a comment saying spark is currently not support but will be added in the future.